smaps/src/main/webapp/css/style.css

```diff
@@ -58,6 +63,12 @@
     box-shadow: 0 0 0 0.2rem rgba(95, 99, 104, .25);
 }
 
+a.btn.btn-gray.active {
+    color: #FFFFFF;
+    background-color: #5F6368;
```

Comment:
Is this file auto-generated?

If not, a comment indicating what color this actually is would be nice.

Reply:
Not auto-generated so yes I can definitely add that.